src/style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  background-color: #e5e5e5;
+  padding: 100px;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  background-color: rgba(255, 255, 255, 1);
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+}
+
+.card__image {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+  height: 134px;
+  margin: 45px auto;
+  margin-top: 29px;
+}
+
+.card__picture {
+  width: 100%;
+}
+
+.card__name-container {
+  margin: 0 15px 6px;
+}
+
+.card__name {
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15.47px;
+}
+
+.card__number-container {
+  margin: 0 15px 15px;
+}
+
+.card__number {
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 12.89px;
+}
+
+.card__feedback {
+  margin: 0 15px 20px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  margin-right: 4px;
+  height: 15px;
+  width: 15px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.card__star--inactive {
+  background-image: url(images/star.svg);
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 12.89px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 15px 20px;
+}
+
+.card__legend {
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 12.89px;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18.75px;
+}
+
+.card__buy {
+  box-sizing: border-box;
+  width: 170px;
+  height: 40px;
+  background-color: rgba(0, 172, 220, 1);
+  border-radius: 5px;
+  margin: 0 15px 15px;
+}
+
+.card__link {
+  text-transform: uppercase;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  text-decoration: none;
+  width: 100%;
+  height: 100%;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 600;
+}
+
+.card__link:hover {
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(0, 172, 220, 1);
```

Comment:
border must be in card__link because when you hover on link, text could change position